extensions/wikia/Lightbox/Lightbox.i18n.php

```diff
@@ -73,11 +73,7 @@
 	'lightbox-share-email-label' => 'Email this image to a friend',
 	'lightbox-share-email-page-label' => 'Email this page to a friend',
 	'lightbox-share-email-page-label-address' => 'Email address',
-	'lightbox-share-email-ok-caption' => 'Confirmation',
 	'lightbox-share-email-ok-content' => '{{PLURAL:$1|Email has|Emails have}} been sent.',
```

Comment:
OK, let's do it this way, but I'm still suspicious. I'm just wondering why these messages are in `Lightbox.i18n.php` and not in something like `SharingToolbar.i18n.php`.

And BTW, links like this one `http://elderscrolls.wikia.com/wikia.php?controller=Lightbox&method=lightboxModalContent&format=html&lightboxVersion=1433238455&userLang=qqx` which are sent as XHRs are broken and should be `http://elderscrolls.wikia.com/wikia.php?controller=Lightbox&method=lightboxModalContent&format=html&lightboxVersion=1433238455&uselang=qqx` (`userLang` → `uselang`). I'll add it to the ticket.